### PR TITLE
Scale selection

### PR DESCRIPTION
**Pull request by d-schindler**
I realised that the current implementation of the scale selection criterion does not recover all local minima of the criterion and that the criterion looks different to my original implementation. Therefore, I replaced the code with my original implementation. In particular, the moving mean is computed with pandas using a triangular window and this seems to be different to the numpy method used previously. However, we should get rid off pandas later.

### CODE REVIEW

File: src/pygenstability/pygenstability.py [hunk 1]
```diff
@@ -173,7 +174,7 @@ def run(
 
         if with_ttprime or with_optimal_scales:
             L.info("Compute ttprimes...")
-            compute_ttprime(all_results, pool)
+            compute_ttprime(all_results, pool, tqdm_disable)
```
Comment (arnaudon) [inline src/pygenstability/pygenstability.py:177]:
```suggestion
            compute_ttprime(all_results, pool)
```

File: src/pygenstability/pygenstability.py [hunk 2]
```diff
@@ -279,7 +280,7 @@ def run_several_louvains(constructor, n_runs, pool):
 
 
 @timing
-def compute_ttprime(all_results, pool):
+def compute_ttprime(all_results, pool, tqdm_disable=False):
```
Comment (arnaudon) [inline src/pygenstability/pygenstability.py:283]:
```suggestion
def compute_ttprime(all_results, pool):
```

**Comment by arnaudon** [comment]
@d-schindler , I'm not sure this works, on the multiscale example it does not as compared to version on master branch. I'll let you fix it before we can merge

**Comment by d-schindler** [comment]
I made a small modification and it should work on our multiscale example. But it would be good to test it on other graphs and compare to the version of @peach-lucien

**Comment by d-schindler** [comment]
as scale selection has become a more central aspect of the whole package, I moved the file from contrib to pygenstability

**Comment by arnaudon** [comment]
@d-schindler , if this goes into the main package, could you add tests to it? thank you!

**Comment by arnaudon** [comment]
ah sorry, as it's called in the main function, it is tested, except the plotting, which we'll add later once the package it more stable, we can merge then!

**Pull request merged**